webapp/channels/src/components/suggestion/at_mention_provider/at_mention_provider.test.tsx

```diff
@@ -359,7 +361,7 @@ describe('components/suggestion/at_mention_provider/AtMentionProvider', () => {
         };
 
         const provider = new AtMentionProvider(params);
-        jest.spyOn(provider, 'getProfilesWithLastViewAtInChannel').mockImplementation(() => [userid10, userid3, userid1, userid2, userid12]);
+        jest.spyOn(provider, 'getProfilesWithLastViewAtInChannel').mockImplementation(() => [userid10, userid3, userid1, userid2]);
```

Comment:
Why are removing here the `userid12` from this list?

Reply:
Oh my bad ! I will add that back ! It was not intensional